src/plugins/touch/README.md

```diff
@@ -0,0 +1,4 @@
+# Touch
+*This documentation here was not authored by the creator of the plugin*
+
+This plugin handles touch input (of mobile devices). You cannot programmatically interact with it. It runs in the background.
```

Comment:
Technically that's not quite true. I'm pretty sure I can send this plugin signals via the event listeners.

I think instead of this section, you could document how to navigate on a touch screen. (by swiping)